tripy/nvtripy/frontend/ops/pooling/avgpool.py

```diff
@@ -115,6 +58,8 @@ def avgpool(
             to apply to the input along each spatial dimension before and after the dimension respectively,
             where :math:`M` is the number of spatial dimensions, i.e. :math:`M = \text{rank(input)} - 2`.
             Defaults to all 0.
+        count_include_pad: Whether to include padded values in the averaging operation. This
+            option must be ``True`` if the input data type is :class:`int8`.
```

Comment:
We can actually shorten this further:
```suggestion
        count_include_pad: Whether to include padded values in the average. 
            Must be ``True`` if the input data type is :class:`int8`.
```